.pre-commit-config.yaml

```diff
@@ -8,3 +8,23 @@ repos:
     hooks:
       - id: end-of-file-fixer
       - id: mixed-line-ending
+
+  - repo: https://github.com/PyCQA/pylint
+    rev: v2.11.1
+    hooks:
+      - id: pylint
```

Comment:
I've been using flake8, and have found it generally better than pylint. Would you mind changing this to flake8? I want to merge it ASAP, and into 0.16.0 branch.

Reply:
I think pylint and flake8 complement each other so I created a separate PR to add flake8 via #484 we can merge which ever one is more important first and then the other one after.